checks/evaluation/permissions/permissions.go

```diff
@@ -132,38 +130,38 @@ func applyScorePolicy(results *checker.TokenPermissionsData, c *checker.CheckReq
 		}
 	}
 
-	if err := reportDefaultFindings(results, c.Dlogger, negativeRuleResults); err != nil {
+	if err := reportDefaultFindings(results, c.Dlogger, negativeProbeResults); err != nil {
 		return checker.InconclusiveResultScore, err
 	}
 
 	return calculateScore(hm), nil
 }
 
 func reportDefaultFindings(results *checker.TokenPermissionsData,
-	dl checker.DetailLogger, negativeRuleResults map[string]bool,
+	dl checker.DetailLogger, negativeProbeResults map[string]bool,
 ) error {
 	// No workflow files exist.
 	if len(results.TokenPermissions) == 0 {
 		text := "no workflows found in the repository"
-		if err := reportFinding("GitHubWorkflowPermissionsStepsNoWrite",
-			text, finding.OutcomeNotApplicable, dl); err != nil {
+		if err := reportFinding("gitHubWorkflowPermissionsStepsNoWrite",
+			text, finding.OutcomeNotAvailable, dl); err != nil {
 			return err
 		}
-		if err := reportFinding("GitHubWorkflowPermissionsTopNoWrite",
-			text, finding.OutcomeNotApplicable, dl); err != nil {
+		if err := reportFinding("gitHubWorkflowPermissionsTopNoWrite",
+			text, finding.OutcomeNotAvailable, dl); err != nil {
```

Comment:
may want to have a `NotApplicable` outcome. Trying to see if it's actually needed when we migrate other checks. Will re-visit.